OracleLinuxDevelopers/oraclelinux8/php/7.4-apache/Dockerfile

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2020 Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+FROM oraclelinux:8-slim
+
+LABEL "org.opencontainers.image.authors"="Oracle Linux Product Team <ol-ovm-info_ww@oracle.com>" \
+      "org.opencontainers.image.url"="https://github.com/oracle/docker-images" \
+      "org.opencontainers.image.source"="https://github.com/oracle/docker-images/tree/master/OracleLinuxDevelopers/oraclelinux8/php/7.4-apache" \
+      "org.opencontainers.image.vendor"="Oracle America, Inc" \
+      "org.opencontainers.image.description"="Oracle Linux 8 (slim) with PHP 7.4 and Apache 2.4.37 installed."
```

Comment:
Will the minor version of httpd remain constant through the module or should this be more generic 2.4.x?

Reply:
Good question. Traditionally, Apache has been held at X.Y.Z versioning and I see no indication that will change within a stream. If we get a new stream, we would have to update the module definition for libdnf anyway.